src/Workspaces/Core/Portable/Remote/RemoteArguments.cs

```diff
@@ -43,29 +43,34 @@ internal class SerializableSymbolAndProjectId : IEquatable<SerializableSymbolAnd
         public ProjectId ProjectId;
 
         public override int GetHashCode()
-            => Hash.Combine(SymbolKeyData, ProjectId.GetHashCode());
+            => SymbolKeyData.GetHashCode();
 
         public override bool Equals(object obj)
             => Equals(obj as SerializableSymbolAndProjectId);
 
         public bool Equals(SerializableSymbolAndProjectId other)
-            => other != null && SymbolKeyData.Equals(other.SymbolKeyData) && ProjectId.Equals(other.ProjectId);
+            => other != null && SymbolKeyData.Equals(other.SymbolKeyData);
```

Comment:
this matches the semantics we had/wanted for SymbolAndProjectId where we unify on the symbol and don't care about hte project.  There's a comment for this in SymbolAndProjectId that i copy over later.